.travis.yml

```diff
@@ -1,15 +1,28 @@
 language: node_js
+
 node_js:
 - '7'
 sudo: false
 os:
 - linux
 services:
   - postgresql
+  - docker
 addons:
   postgresql: "9.4"
+
 before_script:
 - "createdb travis_ci_test"
+- docker run -d -e POSTGRES_DB=travis_ci_test --name postgres postgres:9.4 && sleep 10
```

Comment:
Quick note for future reference: When we wrap in #212, Docker compose should handle this for us.